app/public/css/0-1366px.css

```diff
@@ -0,0 +1,28 @@
+@media screen and (max-height: 1366px) {
```

Comment:
seems like those css files got merged by accident from https://github.com/vacuumlabs/adalite/pull/810 as it wasn't merged yet

Reply:
Yeah, you started reviewing just as I was rebasing it on the css PR, that's all.